.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: 🐛 Bug Report
+about: If something isn't documented correctly 🤔
+
+---
+
+### Current behavior
+
+A clear and concise description of what the bug is.
+
+### Expected behavior/output
+
+A clear and concise description of what you expected to happen.
+
+### Links
+
+One or more links to the documentation page, preferably to the according
+section. If applicable, link also to other resources which might be helpful
+to process your bug report.
+
+### TYPO3 versions
+
+[e.g. 11, 12]
+
+### Possible Solution
+
+Only if you have suggestions on a fix for the bug.
```

Comment:
I would change this: 

> Only if you have suggestions on a fix for the bug.

Only if you have concrete suggestions to solve the problem.

Reply:
Done